app/controllers/api/v1/merchants/random_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Api::V1::Merchants::RandomController < ApplicationController
+
+  def show
+    render json: Merchant.take
```

Comment:
@akintner `take` doesn't really provide a random object, but always returns the same.

`Merchant.all.shuffle.first` would return a different, random object each time it is run. I suggest we update this implementation, calling `all.shuffle.first` through a `random` method within the model.